impl/Cargo.toml

```diff
@@ -34,7 +35,13 @@ serde = "1.0.84"
 toml = "0.4.10"
 serde_json = "1.0.34"
 tempdir = "0.3.7"
+url = "2.0"
+
+cargo = { version = "0.42.0", optional = true }
 
 [dev-dependencies]
 lazy_static = "1.2.0"
 hamcrest2 = "0.3.0"
+
+[features]
+cargo_internals = ["cargo"]
```

Comment:
Does it make sense to make this be default enabled so that it doesn't break existing users?